circus-train-api/pom.xml

```diff
@@ -35,6 +35,12 @@
       <artifactId>metrics-core</artifactId>
     </dependency>
 
+    <dependency>
+      <groupId>com.hotels</groupId>
+      <artifactId>hcommon-ssh</artifactId>
+      <scope>provided</scope>
```

Comment:
Why is the scope for this "provided"? This is saying we expect someone else to have put it on the classpath?

Reply:
this is an error, probably copy&pasta

Reply:
Actually this doesn't matter because is the API, `circus-train-core` provides the library

Reply:
OK, so can you remove it?

Reply:
no, I need the marker for the closeable metastore interface

Reply:
So what does "provided" get you here as opposed to not listing a scope at all?

Reply:
make it not transitive?

Reply:
Yeah, but leaving the scope out would do the same. "provided" you usually use if someone else is going to put the jars on the classpath (as we often do with Hadoop being provided). I just don't understand what "provided" gets us here?

Reply:
the default is compile which makes the dependency transitive, i.e. any downstream implementation of the API will pull the SSH library which they probably don't need. This is all theoretical

Reply:
Yeah, we don't give any other dependency here this special treatment which is why this one stands out. I can live with it I guess.